_docs/collection-details.md

```diff
@@ -145,6 +145,72 @@ Finally once the payment goes through the customer will see a success page:
 
 ![Paga transaction page 4]({{ "/img/collections/paga_mobile_page_4.png" | prepend: site.baseurl }})
 
+# NGN card or bank collection requests through Providus (Beta)
+
+To initiate an NGN bank or card collections through Providus, use the following details:
+
+{% capture data-raw %}
+```javascript
+"input_currency": "NGN",
+"payin_methods": [
+  {
+    "type": "NGN::Bank",
+    "in_details": {
+      "payment_method": "bank", // or "card"
```

Comment:
both of this is optional, so please describe make sure this is noted. Also you can describe below what heppns if you do specify them and what happens if you don't